utils/frame/benchmarking-cli/Cargo.toml

```diff
@@ -16,6 +16,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 chrono = "0.4"
 clap = { version = "3.1.18", features = ["derive"] }
 codec = { package = "parity-scale-codec", version = "3.0.0" }
+comfy-table = "5.0.1"
```

Comment:
```suggestion
comfy-table = { version = "5.0.1", default-features = false }
```

Reply:
worth checking if this can solve the dependency issue with polkadot (tty feature may be needed, not sure).
otherwhise a polkadot companion may be needed to update dependencies.